src/components/shared/FormInfo.js

```diff
@@ -32,54 +40,103 @@ const FormInfo = (props) => {
     nameButton,
   } = props;
 
-  const initialState = useMemo(
-    () => ({
-      title: '',
-      release_date: '',
-      poster_path: '',
-      genres: '',
-      overview: '',
-      runtime: '',
-    }),
-    []
-  );
+  const initialValues = {
+    id: '',
+    title: '',
+    date: '',
+    url: '',
+    genre: '',
+    overview: '',
+    runtime: '',
+  };
 
-  const [selectedOption, setSelectedOption] = useState(null);
+  const FormSchema = Yup.object({
```

Comment:
This object could be moved out of the component to weight off the component size and also to prevent re-initialization of the same object at every re-render.

Reply:
fix